snapteld.go

```diff
@@ -109,7 +109,7 @@ const (
 	defaultLogPath     string = ""
 	defaultLogTruncate bool   = false
 	defaultLogColors   bool   = true
-	defaultConfigPath  string = "/etc/snap/snapd.conf"
+	defaultConfigPath  string = "/etc/snap/snapteld.conf"
```

Comment:
I've made a note to symlink this as well in the packaging since this will be a breaking upgrade.